client/components/back-button/style.scss

```diff
@@ -1,3 +1,5 @@
+@import "calypso/assets/stylesheets/shared/mixins/breakpoints";
```

Comment:
Do those CSS changes intentionally belong to this PR?

Reply:
Yep, those were also failing in the build